README.md

```diff
@@ -564,6 +564,42 @@ let layout = { (view1: UIView, view2: UIView) in
 }
 ```
 
+###Void arguments/return types
+
+It is preferable to not use `Void` in closure arguments or return types, omitting parentheses where possible.
+
+#####Like this:
+```Swift
+let noArgNoReturnClosure = { doSomething() } // no arguments or return types, omit both
+let noArgClosure = { () -> Int in return getValue() } // void argument, use '()'
+let noReturnClosure = { (arg) in doSomething(with: arg) } // void return type, omit return type
+```
+
+#####Not this:
```

Comment:
Please add an addition `#` to the `Not this:` that you've added. (6 total `#` to match visually)